client/src/App.js

```diff
@@ -2,32 +2,36 @@ import React from "react";
 import { MuiThemeProvider } from "@material-ui/core";
 import { BrowserRouter, Route } from "react-router-dom";
 import { theme } from "./themes/theme";
-import LandingPage from "./pages/Landing";
 import SignUp from "./pages/SignUp";
 import Login from "./pages/Login";
 
 import "./App.css";
 import Navbar from "./components/Navbar";
 import ProfileSettings from "./pages/ProfileSettings";
+import ProfileEdit from './components/ProfileEdit';
 
 import { AuthProvider } from "./contexts/user";
 import PrivateRoute from "./components/PrivateRoute/";
 
-
 function App() {
   return (
     <MuiThemeProvider theme={theme}>
       <BrowserRouter>
         <AuthProvider>
           <Navbar />
-          <Route exact path="/" component={LandingPage} />
+          <Route exact path="/"/>
           <Route exact path="/signup" component={SignUp} />
           <Route path="/login" component={Login} />
           <PrivateRoute path="/me" component={ProfileSettings} />
-          <PrivateRoute exact path="/become-a-sitter" component={LandingPage} />
-          <PrivateRoute exact path="/my-sitters" component={LandingPage} />
-          <PrivateRoute exact path="/my-jobs" component={LandingPage} />
-          <PrivateRoute exact path="/messages" component={LandingPage} />
+		  <PrivateRoute path="/me/profile-edit">
```

Comment:
Can you remove this portion as the merge from dev covers this?

Reply:
check it out dan